api/src/main/java/javax/config/Config.java

```diff
@@ -127,9 +130,54 @@
     /**
      * Create a {@link ConfigValue} to access the underlying configuration.
      *
-     * @param key the property key
+     * @param propertyName the property key
+     * @return a {@code ConfigValue} to access the given propertyName
      */
-    ConfigValue<String> access(String key);
+    ConfigValue<String> access(String propertyName);
```

Comment:
I would change the method name to `getConfigValue` instead

Reply:
I'd rather not do that. Then we'd have getValue and getConfigValue. That would confuse users I fear.